lib/css/CssModulesPlugin.js

```diff
@@ -401,16 +400,41 @@ class CssModulesPlugin {
 			try {
 				const codeGenResult = codeGenerationResults.get(module, chunk.runtime);
 
-				const s =
+				let s =
```

Comment:
Small nitpick but I think important: Lets use `source`/`generatedSource`/`anyOtherReadableName` instead of `s`.